docs/source/packages/fetch.md

```diff
@@ -57,7 +57,7 @@ async function postData (url, data) {
       const data = await response.json();
       return response(null, data);
     } catch (err) {
-      return response(err, null);
+      return response.status(500).send({status: false, msg: err.message});
```

Comment:
I do not recall the response function having a status or send methods. This looks like Express thing.

Reply:
Yeah sir, you are right. I will look forward to it sir. Thank you.